app/client/widgets/MentionTextBox.ts

```diff
@@ -26,6 +26,21 @@ import {
 const testId = makeTestId('test-mention-textbox-');
 const t = makeT('MentionTextBox');
 
+
+/**
+ * Some not-so-old Firefox ESR versions (<= 128) don't support "plaintext-only" as contenteditable value.
+ *
+ * We make sure to check for support before using it.
+ * If it's not supported, we handle ourselves stripping pasted text formatting.
+ * @see https://stackoverflow.com/a/18316972
+ */
+const supportsPlaintextEditables = () => {
+  const div = document.createElement('div');
+  div.setAttribute('contenteditable', 'PLAINTEXT-ONLY');
+  return div.contentEditable === 'plaintext-only';
+};
+const contentEditableValue = supportsPlaintextEditables() ? 'plaintext-only' : 'true';
```

Comment:
I'd prefer this dom manipulation happen on first use rather than at import time. I.e. something like this?
```
let _contentEditableValue;
function contentEditableValue() {
  return _contentEditableValue || (_contentEditableValue = supportsPlaintextEditables() ? 'plaintext-only' : 'true');
}
```